src/csharp/build_packages_dotnetcli.sh

```diff
@@ -32,6 +32,8 @@ mkdir -p protoc_plugins
 cp -r $EXTERNAL_GIT_ROOT/platform={windows,linux,macos}/artifacts/protoc_* protoc_plugins || true
 # Kokoro flow
 cp -r $EXTERNAL_GIT_ROOT/input_artifacts/protoc_* protoc_plugins || true
+find protoc_plugins/ -type f -name protoc -exec chmod +x {} ";"
```

Comment:
I'm forgetting if the windows or linux package script is the one that bundles up the ofificial packages, but I think this might be needed for the `build_packages_dotnetcli.bat` too

Reply:
The .bat file will not face this issue, because Windows does not have the concept of executable flag on files